### DISCUSSION TITLE
Help finding where extra maven repos configured

### DISCUSSION THREAD

**Discussion by paulrobinson** [jbangdev/jbang#804] [Q&A] [answered]
Hello,

A while ago I created: https://github.com/paulrobinson/jira-field-usage/blob/main/run.java.

It imports `com.atlassian.jira:jira-rest-java-client-api:3.0.0` and `com.atlassian.jira:jira-rest-java-client-core:3.0.0` which I think comes from an Atlassian maven repo. However, I have not set a //REPOS in the run.java.

The following command works for me on my machine, but not for others.

    jbang https://github.com/paulrobinson/jira-field-usage/blob/main/run.java  --jira-server https://issues.redhat.com -u '<jira_username>' -p '<jira_password>' --project QUARKUS

I forget how I made JBang find the Atlassian dependencies. I don't have a `~/.m2/settings.xml`. Can you provide some pointers as to where it might be set? I'd like to remove the global setting and move it into the script, so that it's self-contained and works for others. 

Thanks,

Paul.

**Comment by maxandersen** [comment, marked as answer]
my first guess is that you are using older jbang which use jcenter by default as opposed to mavencentral.

did that change when it was announced jcenter is going away.

To make your script same as before add `//REPOS jcenter, mavencentral` to have it explicitly use jcenter and seocndary mavencentral. you might not need mavencentral though.

**Reply by maxandersen** [reply]
just verified that just adding //REPOS jcenter then dependencies get resolved.

**Reply by maxandersen** [reply]
btw. looks like https://packages.atlassian.com/maven/repository/public is the atlassian backed location to get things from

**Reply by paulrobinson** [reply]
Thanks Max. To make this work after jcenter disappears, I understand we should specify the https://packages.atlassian.com/maven/repository/public repo in the //REPOS bit?

**Reply by maxandersen** [reply]
yeah seems like that.

**Reply by paulrobinson** [reply]
Thanks. We've set it to the following, and it works:

    //REPOS mavencentral,atlassian=https://packages.atlassian.com/maven/repository/public

**Comment by maxandersen** [comment]
oh and update your jbang :)

**Reply by paulrobinson** [reply]
Done, I can now reproduce the problem.
